test/db/cmd/cmd_ab

```diff
@@ -0,0 +1,75 @@
+NAME=ablt
+FILE=bins/mach0/mac-ls2
+CMDS=<<EOF
+e anal.calls=false
+af
+echo $(ablq)
+abt
+ablt fail/str/0x,addr/cols/size/jump/calls
+abl~?
+aflc
+aa
+abl~?
+aflc
+EOF
+EXPECT=<<EOF
+0x1000011e8 0x100001211 0x100001216 0x100001232 0x10000124d 0x100001252 0x10000125c 0x100001276 0x10000127e 0x100001284 0x100001290 0x1000012a8 0x1000012b6 0x1000012bf 0x1000012c6 0x1000012dc 0x1000012e3 0x1000012e7 0x100001301 0x10000130a 0x100001321 0x10000132a 0x100001347 0x100001353 0x100001370 0x100001392 0x100001396 0x1000013a2 0x1000013b1 0x1000013bd 0x1000013cc 0x1000013fb 0x10000140a 0x10000141a 0x10000143f 0x10000144b 0x100001466 0x100001473 0x10000147f 0x100001492 0x1000014a9 0x1000014b4 0x1000014ba 0x1000014d0 0x1000014df 0x1000014f0 0x1000014ff 0x100001519 0x100001525 0x10000153e 0x10000155b 0x10000156a 0x100001572 0x100001583 0x1000015a0 0x1000015ac 0x1000015c9 0x1000015e4 0x1000015f7 0x10000160d 0x10000162a 0x100001636 0x100001645 0x100001665 0x100001682 0x10000168b 0x1000016aa 0x1000016b8 0x1000016cd 0x1000016f1 0x100001768 0x10000177f 0x100001789 0x100001795 0x10000179a 0x1000017a6 0x1000017af 0x1000017b4 0x1000017fe 0x100001875 0x10000187e 0x10000188b 0x1000018c2 0x1000018cb 0x1000018d1 0x1000018dd 0x1000018e3 0x1000018ef 0x1000018f8 0x100001901 0x10000190a 0x100001913 0x10000191c 0x100001925 0x10000192e 0x100001937 0x100001940 0x100001949 0x100001952 0x10000195b 0x100001964 0x10000196d 0x100001976 0x10000197f 0x100001988 0x10000198f 0x1000019a2 0x1000019ab 0x1000019b4 0x1000019bd 0x1000019c6 0x1000019cf 0x1000019d6 0x1000019e2 0x1000019f0 0x1000019f7
+.----------------------------------------------------------------------------------------------------------.
+| addr         | size  | traced  | ninstr  | jump         | fail         | fcns          | calls  | xrefs  |
+)----------------------------------------------------------------------------------------------------------(
+| 0x1000011e8  | 41    | 0       | 14      | 0x100001216  | 0x100001211  |  0x1000011e8  |        |        |
```

Comment:
Better also add a test where xrefs is not empty to get some coverage there.

Reply:
done